src/content/en/fundamentals/primers/service-workers/index.md

```diff
@@ -273,13 +273,7 @@ response of the fetch request and then adding it to the cache, like below.
               return response;
             }
 
-            // IMPORTANT: Clone the request. A request is a stream and
```

Comment:
@jakearchibald - can you confirm this and either request changes or approve the PR?

Reply:
Jake has confirmed the changes are correct.